webapp/channels/src/components/cloud_preview_modal/preview_modal_content.scss

```
@@ -0,0 +1,151 @@
+.preview-modal-content {
+    display: flex;
+    height: 100%;
+    flex-direction: column;
+
+    &__sku-label {
+        display: flex;
+        width: fit-content;
+        align-items: center;
+        padding: 2px var(--spacing-xxxs, 4px);
```

I don't think we have these spacing variables in the codebase yet. Plus it should be 6px left/right padding

```suggestion
        padding: 2px 6px;
```